tests/test_custom_return_types.py

```diff
@@ -276,3 +276,251 @@ def test_hash_invalid_type(self):
         res = self.dag.weighted_edge_list()
         with self.assertRaises(TypeError):
             hash(res)
+
+
+class TestPathMapping(unittest.TestCase):
+
+    def setUp(self):
+        self.dag = retworkx.PyDAG()
+        node_a = self.dag.add_node('a')
+        self.dag.add_child(node_a, 'b', "Edgy")
+
+    def test__eq__match(self):
+        self.assertTrue(
+            retworkx.dijkstra_shortest_paths(self.dag, 0) == {1: [0, 1]})
```

Comment:
The existing dijkstra tests would also exercise this now, for larger graphs, correct?

Reply:
Yeah, I just added it here to have an explicit test of the python comparison api for `__eq__`